eslint.config.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const eslintConfigESLint = require("eslint-config-eslint");
```

Comment:
```suggestion
const eslintConfigESLint = require("eslint-config-eslint/cjs");
```

Since this is a CommonJS project, it should use the config for CommonJS projects.